packages/infra/config/env-config.ts

```diff
@@ -24,7 +24,7 @@ export type EnvConfig = {
   authSubdomain: string; // Authentication subdomain
   dbName: string;
   dbUsername: string;
-  loadBalancerDnsName?: string;
+  loadBalancerDnsName: string;
```

Comment:
optional -> required since Bulk signing Lambda now needs it

Reply:
It was optional b/c the first time the API gets provisioned this is not known. There's a cyclical dependency there, that sucks - ideally, we would use pre-defined internal URLs, like `api.staging.metriport.com` or `api.internal.staging.metriport.com`, but we didn't do this in the beginning.

Spinning off a new env would look like this:
- deploy API Stack (that lambda doesn't get created);
- update env config with the LB address;
- deploy again (lambda gets created).

If we make this required, we can't deploy the API Stack on new envs.